src/store/actions.ts

```diff
@@ -210,6 +211,24 @@ export async function loadTokensList(state: State): Promise<State> {
     return newState;
 }
 
+export async function loadConversionHistory(state: State, account: string = undefined): Promise<State> {
+    const newState = { ...state };
+
+    try {
+        var conversionSentReceived = await loadConversionSentReceived(account);        
```

Comment:
Just a minor thing, but if we can use const and let in place of var, that would be awesome.

Reply:
Done!